modules/apps/31-ibc-query/types/msgs.go

```diff
@@ -61,13 +76,13 @@ func NewMsgSubmitCrossChainQueryResult(id string, result QueryResult, data []byt
 }
 
 // ValidateBasic implements sdk.Msg and performs basic stateless validation
-func (msg MsgSubmitCrossChainQueryResult) ValidateBasic() error {
+func (q MsgSubmitCrossChainQueryResult) ValidateBasic() error {
 	return nil
 }
 
 // GetSigners implements sdk.Msg
-func (msg MsgSubmitCrossChainQueryResult) GetSigners() []sdk.AccAddress {
-	signer, err := sdk.AccAddressFromBech32(msg.Relayer)
+func (q MsgSubmitCrossChainQueryResult) GetSigners() []sdk.AccAddress {
+	signer, err := sdk.AccAddressFromBech32(q.Relayer)
```

Comment:
I think "sender" is more natrual than "relayer".
Although we design that only relayer use this transaction, but it can be changed later. And newbie may not understand why the transaction has attribute "relayer"

This part is not inserted in your commit, but could you change this part please? sorry..